core/domain/classifier_services.py

```diff
@@ -153,6 +153,75 @@ def handle_non_retrainable_states(exploration, state_names, exp_versions_diff):
         job_exploration_mappings)
 
 
+FLOAT_INDICATOR_KEY = 'float_values'
+# pylint: disable=too-many-branches
```

Comment:
Put this within the function, under the function's docstring.

Reply:
Done.